django/template/base.py

```diff
@@ -151,6 +151,9 @@ def __eq__(self, other):
             self.loader == other.loader
         )
 
+    def __ne__(self, other):
```

Comment:
I personally don't think that's necessary. A grep for __ne__ and similar Python 2 methods on the entire Django codebase will find this at that time in the future. But if you want to add it I'm fine with it :)